pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -268,7 +267,7 @@ func (i *CommandInitOption) makeKarmadaKubeControllerManagerDeployment() *appsv1
 					fmt.Sprintf("--cluster-name=%s", options.ClusterName),
 					fmt.Sprintf("--cluster-signing-cert-file=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
 					fmt.Sprintf("--cluster-signing-key-file=%s/%s.key", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
-					"--controllers=namespace,garbagecollector,serviceaccount-token",
+					"--controllers=namespace,garbagecollector,serviceaccount-token,ttl-after-finished,bootstrapsigner,tokencleaner,csrapproving,csrcleaner,csrsigning",
```

Comment:
Are these controllers are [required controllers](https://karmada.io/docs/administrator/configuration/configure-controllers#required-controllers) or [optinal controllers](https://karmada.io/docs/administrator/configuration/configure-controllers#optinal-controllers)?

Can you explain why they are needed for this feature?

Reply:
These controllers are exclusively used by new register workflow. If there aren't the controllers, the new register process prossiblly will be not executed.

I think they are optinal, only enabling new register process need them.

> Can you explain why they are needed for this feature?

bootstrapsigner: be used to sign csr automatically.
tokencleaner: be used to clean the tokens that are expired automatically.
csrapproving,csrcleaner,csrsigning: enable csr function.

I will also explain them in the document after this pr.